packages/website/blog/2024-04-30-announcing-typescript-eslint-v8-beta.md

```diff
@@ -0,0 +1,226 @@
+It's been experimentally available since v6.1.0 under the name `EXPERIMENTAL_useProjectService`; now, we've renamed it to `projectService`.
+
+You can use the new project service in your configuration instead of the previous `parserOptions.project`:
+
+```js title="eslint.config.js"
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  eslint.configs.recommended,
+  ...tseslint.configs.recommendedTypeChecked,
+  {
+    languageOptions: {
+      parserOptions: {
+        // Remove this line
+        project: true,
+        // Add this line
+        projectService: true,
+        tsconfigRootDir: import.meta.dirname,
+      },
+    },
+  },
+);
+```
+
+The project service will automatically find the closest `tsconfig.json` for each file (like `project: true`).
+It also allows enabling typed linting for files not explicitly included in a `tsconfig.json`.
+This should remove the need for custom `tsconfig.eslint.json` files to lint files like `eslint.config.js`!
+
+```js title="eslint.config.js"
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  eslint.configs.recommended,
+  ...tseslint.configs.recommendedTypeChecked,
+  {
+    languageOptions: {
+      parserOptions: {
+        // Remove this line
+        project: ['packages/*/tsconfig.json', 'tsconfig.eslint.json'],
+        // Add this line
+        projectService: {
+          allowDefaultProjectForFiles: ['./*.js'],
+          defaultProject: './tsconfig.json',
+        },
+        tsconfigRootDir: import.meta.dirname,
+      },
+    },
+  },
+);
+```
+
+Internally, the project service uses the same TypeScript APIs that editors such as VS Code use.
+Doing so should make it harder to accidentally configure different type information for ESLint than what you see in day-to-day editing.
+
+We're thrilled to have this option promoted to stable in v8.
+We'll soon release a [dedicated `parserOptions` blog post](https://github.com/typescript-eslint/typescript-eslint/pull/8031) walking through the new option in more details.
+
+### Updated Configuration Rules
+
+Every new major version of typescript-eslint comes with changes to which rules are enabled in the preset configurations - and with which options.
+Because this release also includes a reworking of the configurations themselves, the list of changes is too large to put in this blog post.
+Instead see the table in [Changes to configurations for 8.0.0](https://github.com/typescript-eslint/typescript-eslint/discussions/8914) for a full list of the changes.
+
+Please do try out the new rule configurations presets and let us know in that discussion!
+
+:::tip
+If your ESLint configuration contains many `rules` configurations, we suggest the following strategy to start anew:
```

Comment:
What's the motivation behind this section? I'm just thinking, the people have many custom configurations are probably not the people who would want to nuke their custom config and use the defaults.

Reply:
IME what happens to a _lot_ of teams is that they start off with clean configs, then slowly make them more and more convoluted over the years. They then get in the mindset of _"this is all valuable, we shouldn't remove any of it"_. Having this explicitly encourage them to rethink helps break that mindset. Plus devs tend to do a lot more docs reading when they (re-)make a file from scratch - they feel more ownership over it.